models/ner_model.py

```diff
@@ -1,10 +1,12 @@
-from .training_model_api import TrainingModel
```

Comment:
These should stay as relative imports I think? Because when you import NerModel from gui_v0 in a notebook, you'd want these to work. 
e.g.
`from gui_v0.models.ner_model import NerModel` won't work if it says `from models import ...`
I might be wrong but it didn't work when I tried it in jupyter in the server.

Reply:
Oh weird. The relative imports weren't working for me when I was developing locally or on the server so I had to write models.<<whatever>>. Maybe we could just do absolute from gui_v0 everywhere?